src/plugins/orangehrmOpenidAuthenticationPlugin/config/routes.yaml

```
@@ -15,17 +15,23 @@ apiv2_auth_openid_provider:
 ##########################
 #        Screens
 ##########################
-add_auth_provider:
+admin_add_auth_provider:
   path: admin/addAuthProvider
   controller: OrangeHRM\OpenidAuthentication\Controller\SaveSocialMediaAuthProviderController::handle
   methods: [ GET ]
 
 edit_auth_provider:
```

```suggestion
admin_edit_auth_provider:
```
i assumed you will fix this as well